SCOPES.md

```diff
@@ -0,0 +1,45 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style balrog tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to Schedule a release to ship on balrog channel(s)
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidate-tasks`
+
+* `{scope_prefix}:action:push-to-release`
+  * Tells beetmoverscript to submit locale information for a release.
+    * This is **default** if no `action` specified in scopes.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-push-tasks`
+
+* `{scope_prefix}:action:push-to-nightly`
+  * Tells beetmoverscript to submit the initial toplevel release information.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-nightly-tasks`
+
```

Comment:
There is also a `push-to-partner` missing here. It conflicts with any other action and "Tells beetmoverscript to move the corresponding artifacts of partner releases of Firefox to their corresonding private buckets"